examples/cloud_run/README.md

```diff
@@ -1,5 +1,7 @@
 # Deploy a function to Cloud Run
 
+[![Run on Google Cloud](https://deploy.cloud.run/button.svg)](https://deploy.cloud.run)
```

Comment:
This is a subdirectory, do we need the `dir` param or is that inferred?

Reply:
It should be inferred.